src/optest/opexpect.py

```diff
@@ -85,7 +85,8 @@ def handle_opal_err(pty, is_assert):
 
     # hmm,  might need to move this elsewhere
     l = super(spawn, pty).expect(["boot_entry.*\r\n",
-                                   "Initiated MPIPL", pexpect.TIMEOUT],
+                                   "Initiated MPIPL",
+                                   "NVRAM: Failed to load", pexpect.TIMEOUT],
```

Comment:
?

Reply:
The test is expected failure, However still is looking for the string and the last string is added to expected string.